### ISSUE TITLE
[COGNITION]: VAOS Scheduler - Title attribute must update on every route change

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2843]
## Description

The WCAG specification says a web pages should have a title that describes topic or purpose. From the spec:
> This criterion benefits all users in allowing users to quickly and easily identify whether the information contained in the Web page is relevant to their needs.

Our VAOS application uses the same `<title>` on every view. This should update to include the `<h1>` string and any VAOS or VA.gov specific information the content team recommends.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a user, I want the title to update and my tab in modern browsers to show me the new information.

## Possible Fixes (optional)

I've tried out [React Helmet](https://github.com/nfl/react-helmet) before on the AnyCorp learning app, and it worked very well, was easy to implement. There's also an option to update `document.title` using `componentDidMount()` lifecycle method.

## WCAG or Vendor Guidance (optional)

* [Page Titled: Understanding SC 2.4.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-29 at 9 08 27 AM](https://user-images.githubusercontent.com/934879/67774937-3c60e980-fa2c-11e9-850f-e78b8e963aa1.png)
